R/declare_estimator.R

```diff
@@ -261,6 +261,18 @@ tidy_estimator <- function(fn) {
   f
 }
 
+#' @rdname ourPkg-deprecated
+#' @section \code{tidy_estimator}:
+#' For \code{tidy_estimator}, use \code{\link{label_estimator}}.
+#'
+#' @importFrom lifecycle deprecate_soft
+#'
+#' @export
+tidy_estimator <- function(estimator_function) {
+  deprecate_soft("1.0.0", "tidy_estimator()", "label_estimator()")
```

Comment:
Seems fine, but I'd recommend just using message/warn/stop/.Deprecated from base R instead of pulling in an extra tidyverse library just for a couple messages, especially if the intent is to remove them further down the road. Also this will cause the 1-2 designs in DesignLibrary that were using this to start throwing warnings, CRAN will probably flag that when you submit.